### DISCUSSION TITLE
Public generic Reaction Database?

### DISCUSSION THREAD

**Discussion by kienerj** [rdkit/rdkit#6883] [Q&A]
Does anyone here know of a public "generic" reaction database with generic meaning SMIRKS or SMARTS based or rxn-files with according generalizations (like when drawn in Marvin).

**Comment by dehaenw** [comment]
This may be of some use: rdkit compatible datamol has 127 patterns for some common reactions: https://github.com/datamol-io/datamol/blob/9e94d026534b2a534250dfbfab924ab6f089e477/datamol/data/reactions.json
